Source/Core/DolphinQt2/MainWindow.cpp

```diff
@@ -2,11 +2,12 @@
 // Licensed under GPLv2+
 // Refer to the license.txt file included.
 
+#include <QApplication>
 #include <QDir>
 #include <QFileDialog>
 #include <QIcon>
 #include <QMessageBox>
-
+#include <qpa/qplatformnativeinterface.h>
```

Comment:
It does warn about it, but as I see it there is no alternative with the current system.